content/docs/components/tabs.md

```diff
@@ -76,138 +78,26 @@ Work in progress. At a mobile resolution (below 768px) an active tab will collap
   </div>
 </div>
 {{< /highlight >}}
-</div>
-
-
```

Comment:
Assuming these examples will eventually go into Launch Pad somewhere?

Reply:
To clarify: `tabs--flat` example. I think some sort of visual is still nice for some of these examples.

Reply:
That's the idea!